packages/celotool/src/e2e-tests/utils.ts

```diff
@@ -438,6 +436,22 @@ export async function initAndStartGeth(gethBinaryPath: string, instance: GethIns
   return startGeth(gethBinaryPath, instance)
 }
 
+// Add validator 0 as a peer of each other validator.
+async function connectValidatorPeers(gethConfig: GethTestConfig) {
+  const admins = gethConfig.instances
+    .filter(({ wsport, rpcport }) => wsport || rpcport)
+    .map(
+      ({ wsport, rpcport }) =>
+        new Admin(`${wsport ? 'ws' : 'http'}://localhost:${wsport || rpcport}`)
+    )
+  const enode = (await admins[0].getNodeInfo()).enode
+  await Promise.all(
+    admins.slice(1).map((admin) => {
+      admin.addPeer(enode)
```

Comment:
I think you might need to return here or not wrap this function call with `{}` in order for Promise.all to work as intended

Reply:
Thanks!